planning/behavior_velocity_planner/src/scene_module/occlusion_spot/occlusion_spot_utils.cpp

```diff
@@ -246,17 +245,25 @@ ArcCoordinates getOcclusionPoint(const PredictedObject & obj, const ConstLineStr
   return arcs.at(0);
 }
 
-// calculate value removing offset distance or 0
-double calcSignedLateralDistanceWithOffset(const double lateral, const double offset)
+// calculate lateral distance value to collision point of each object.
+double calcSignedLateralDistanceWithOffset(
```

Comment:
@hasanzfdn 
Thank you fot this change but it was a bit hard to read if new change was equal to previous.
Can you refactor this part like this?
```
// calculate lateral distance value to collision point of each object.
double calcSignedLateralDistanceWithOffset(
  const double lateral, const double right_overhang, const double left_overhang,
  const double wheel_tread)
{
  const double offset_left = left_overhang + wheel_tread / 2;
  const double offset_right = right_overhang + wheel_tread / 2;
  if (lateral > 0) {
    return std::max(lateral + offset_left, 0.0);
  }
  // else
  return std::min(lateral - offset_right, 0.0);
}
```